Makefile

```diff
@@ -406,11 +406,15 @@ cluster/cleanup/serviceaccount: kustomize
 	@-$(KUSTOMIZE) build config/rbac-$(INSTALLATION_SHORTHAND) | oc delete -f -
 
 .PHONY: cluster/cleanup/olm
-cluster/cleanup/olm: cluster/cleanup
+cluster/cleanup/olm: cluster/cleanup cluster/cleanup/subscription
```

Comment:
I wonder do we need to cleanup the subscription as the namespace is deleted in `cluster/cleanup` :thinking:

Reply:
I've been wondering this myself and tried just running an old version of the `make cluster/cleanup/olm` subscription still was there :thinking: It sounds strange. Maybe I wasn't patient enough to wait for the subscription to be deleted. I will double-check this and will be back with more that my vague memories

Reply:
Yep, the subscription is gone when the namespace is deleted. Removed changes to the makefile. Thanks